Fabric.Authorization.Persistence.SqlServer/EntityModels/Group.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using Fabric.Authorization.Domain.Models;
+
+namespace Fabric.Authorization.Persistence.SqlServer.EntityModels
+{
+    public class Group : ITrackable, ISoftDelete
+    {
+        public Group()
+        {
+            Roles = new List<Role>();
+            //Users = new List<User>();
+        }
+        public int Id { get; set; }
+        public Guid ExternalIdentifier { get; set; }
```

Comment:
I think we set this value to GroupName in the API, so I don't think it can be a Guid.

Reply:
I've addressed this.